visualize_of_landingMining_sites/README.md

```diff
@@ -0,0 +1,17 @@
+#  Deep Neural Network with H20-R: Satellite-Image Classification 1
```

Comment:
I'm not sure what this README has to do with your project.  Please delete and rewrite.